tests/quantization/bnb/test_4bit.py

```diff
@@ -267,15 +273,18 @@ def test_device_assignment(self):
         self.assertEqual(self.model_4bit.device.type, "cpu")
         self.assertAlmostEqual(self.model_4bit.get_memory_footprint(), mem_before)
 
-        # Move back to CUDA device
-        self.model_4bit.to(0)
-        self.assertEqual(self.model_4bit.device, torch.device(0))
-        self.assertAlmostEqual(self.model_4bit.get_memory_footprint(), mem_before)
+        if torch.cuda.is_available():
+            # Move back to CUDA device
+            self.model_4bit.to(0)
+            self.assertEqual(self.model_4bit.device, torch.device(0))
```

Comment:
Should these be
```suggestion
            self.model_4bit.to("cuda")
            self.assertEqual(self.model_4bit.device, torch.device("cuda"))
```
for clarity?